src/migrations/20231030150724-sim_score_cache.js

```diff
@@ -0,0 +1,55 @@
+const {
+  prepMigration,
+} = require('../lib/migration');
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      const sessionSig = __filename;
+      await prepMigration(queryInterface, transaction, sessionSig);
+      await queryInterface.createTable(
+        'SimScoreCaches',
+        {
+          id: {
+            type: Sequelize.INTEGER,
+            allowNull: false,
+            primaryKey: true,
+            autoIncrement: true,
+          },
+          recipient_id: { allowNull: false, type: Sequelize.INTEGER },
+          goal1: {
+            allowNull: false,
+            type: Sequelize.INTEGER,
+            default: null,
+            references: {
+              model: {
+                tableName: 'Goals',
+              },
+            },
+          },
+          goal2: {
+            allowNull: false,
+            type: Sequelize.INTEGER,
+            default: null,
+            references: {
+              model: {
+                tableName: 'Goals',
+              },
+            },
+          },
+          score: { allowNull: false, type: Sequelize.DECIMAL(3, 1) },
+          createdAt: { allowNull: false, type: Sequelize.DATE },
```

Comment:
can we have a column for ignore bool. or should this be a migration onto this table later?

Reply:
Maybe we should wait on this one until we've figured out this logic a bit more. If it's as simple as we think it will be, then the migration will be extremely quick and easy -- but maybe that won't be the case.